tests/orion_config.yaml

```diff
@@ -0,0 +1,4 @@
+database:
+    type: 'mongodb'
+    name: 'orion_test'
+    host: 'mongodb://user:pass@localhost'
```

Comment:
Is there a way to bring a more general conftest higher up at `tests/` level to provide these fixtures that are universally used throughout both `functional` and `unittests`?